### PR TITLE
RDAPI-29320 Validate user expiry field

### PR DESCRIPTION
**Pull request by CillianLong**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes
Description added for the new field in API Manager Settings named Validate user expiry 

## Deploy preview link
http://pr-2599.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_reference/api_mgmt_config_web/#apiportal

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | d86a312 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2599.axway-open-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/apim_reference/api_mgmt_config_web.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ For more details on API Portal, see [Administer API Portal](/docs/apim_adminis
 
 * **User registration**: Select whether to enable automatic user registration. This is enabled by default.
 * **Auto-approve user registration**: Select whether automatic approval of user registration requests is enabled. This is enabled by default.
+* **User Registration link expiry**: A registration email is sent to a 3rd party application developer with a link to activate their account. This will expire if it has not been activated after the set time period, making the activation invalid. The default value is 1440 mins (24 hours). The maximum value is 14400 mins (10 days). Setting this value to 0 will disable this requirement.
```
Comment (andreamussap) [inline content/en/docs/apim_reference/api_mgmt_config_web.md:68]: (User Registration link expiry) - is this capitalization correct, consistent with the UI?

When you clarify this, can you please make some minor improvements to the paragraph, something like:

A registration email is sent to the user to activate their account. The default activation value is 1440 mins (24 hours), starting when the email is sent, and the maximum value is 14400 mins (10 days).

After the set time period has elapsed, the activation becomes invalid and you must (do something to get another email to get your account activated ??)

Setting the value of this property to `0` disables the requirement for the user to activate their account.

**Comment by andreamussap** [comment]
@CillianLong @kmercierwhite  If the '**User registration**' is enabled by default and the '**User Registration link expiry**' is also enabled by default, isn't this a conflict, like, user registration is enabled by default, but the user also receives an email to activate their account by default, but the account is already activated by default... it's confusing
 
 http://pr-2599.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_reference/api_mgmt_config_web/#general-settings

**Comment by CillianLong** [comment]
> @CillianLong @kmercierwhite If the '**User registration**' is enabled by default and the '**User Registration link expiry**' is also enabled by default, isn't this a conflict, like, user registration is enabled by default, but the user also receives an email to activate their account by default, but the account is already activated by default... it's confusing
> 
> http://pr-2599.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_reference/api_mgmt_config_web/#general-settings

Hi @andreamussap the 'User Registration' field refers to the register link on the sign in page, and whether it is displayed or not. The 'User Registration link expiry field' refers to the expiry period of the activation link sent by the resulting email.

**Pull request closed without merging**